tests/integration/test_fim/test_registry/test_tags/test_registry_tags.py

```diff
@@ -6,38 +6,31 @@
 import pytest
 
 from wazuh_testing import global_parameters
-from wazuh_testing.fim import LOG_FILE_PATH, regular_file_cud, generate_params, delete_registry, registry_parser
+from wazuh_testing.fim import LOG_FILE_PATH, registry_value_cud, generate_params, delete_registry, registry_parser
 from wazuh_testing.tools import PREFIX
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.tools.monitoring import FileMonitor
-from win32api import RegOpenKeyEx
 import win32con
 
-pytestmark = [pytest.mark.win32, pytest.mark.tier(level=0)]
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=1)]
 
 # Variables
 key = "HKEY_LOCAL_MACHINE"
-sub_key_1 = "SOFTWARE\\Classes\\testkey"
-sub_key_2 = "SOFTWARE\\testkey"
+sub_key = "SOFTWARE\\test_key"
+sub_key_2 = "SOFTWARE\\Classes\\test_key"
 
-handle_list = list()
-test_regs = [os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)]
-registry_str = ",".join(test_regs)
-wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-registr_str, registry2 = test_regs
-
-registry_list=[(sub_key_1, win32con.KEY_WOW64_64KEY), (sub_key_2, win32con.KEY_WOW64_32KEY),
-               (sub_key_2, win32con.KEY_WOW64_64KEY)]
-
-monitoring_modes = ['scheduled', 'scheduled']
-
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_regs = [ os.path.join(key, sub_key), os.path.join(key, sub_key_2)]
```

Comment:
Remove whitespace after `[`.
```suggestion
test_regs = [os.path.join(key, sub_key), os.path.join(key, sub_key_2)]
```